packages/ui/src/components/Icon.svelte

```diff
@@ -34,12 +34,15 @@
   $: url = isAsset(icon) ? getMetadata(toAsset(icon)) ?? 'https://anticrm.org/logo.svg' : ''
 
   $: _fill = iconProps?.fill ?? fill
+
+  // workaround to prevent sveltejs bug in `svelte:component`: https://github.com/sveltejs/svelte/issues/9177
```

Comment:
That seem to be quite a major bug that might affect not only icons but potentially all places where we use `svelte:component`. How we can ensure those are not affected?

Reply:
Yes, we need to check all places where use use svelte:component and address them until issue will be fixed.

Reply:
Does svelte:self or svelte:fragment also affected?

Reply:
I've just found https://github.com/sveltejs/svelte/pull/10604, and we probable should update to 
[svelte@4.2.12](https://github.com/sveltejs/svelte/releases/tag/svelte%404.2.12) to address it instead.

Reply:
Yeah, let's update to 4.2.12 and check if the issue is fixed.

Reply:
I will update